### DISCUSSION TITLE
连不上站点（？）

### DISCUSSION THREAD

**Discussion by Kauta1** [nonebot/discussions#147] [问答] [answered]
An error occurred while requesting https://cdn.jsdelivr.net/gh/nonebot/nonebot2/website/static/drivers.json.
安装插件的时候都会出现这一小行红字
![image](https://user-images.githubusercontent.com/105104706/169046053-4b7430df-b547-4606-8df3-db39ba80e65b.png)

**Comment by yanyongyu** [comment, marked as answer]
jsdelivr cdn在中国区出现了大面积连接问题

**Reply by Kauta1** [reply]
谢谢！
